src/Controls/src/Core/SafeAreaGroup.cs

```diff
@@ -0,0 +1,28 @@
+#nullable disable
+using System;
+
+namespace Microsoft.Maui.Controls
+{
+	/// <summary>
+	/// Specifies which platform safe area insets to ignore for a layout or visual element.
+	/// </summary>
+	[Flags]
+	public enum SafeAreaGroup
+	{
+		/// <summary>
+		/// Apply platform inset - content will be positioned only in the safe area.
+		/// </summary>
+		None = 0,
+
+		/// <summary>
+		/// Ignore all insets for this edge - content may be positioned anywhere on the screen, 
+		/// including behind toolbars, screen cutouts, etc.
+		/// </summary>
+		All = 1 << 0,
```

Comment:
Should all be if all bits of the flag are one? 

Make sure this flag is future proofed for when we add additional properties